File: Modul-3/Soal 5/PRAK305-2410817210011-MUHAMMADDZULFATHIAHYAN.c
#include <stdio.h>
int main()
{
    int input, hari, jam, menit, detik;
    printf("");
    scanf("%d",&input);
    hari =(input / 86400);
    jam = (input % 86400) / 3600;
    menit = (input % 3600) / 60;
    detik = (input % 60);
    if (hari > 0){printf("%d hari %02d:%02d:%02d\n", hari, jam, menit, detik);}
    else {printf("%02d:%02d:%02d\n", jam, menit, detik);}
    return 0;
}

File: Modul-3/Soal 5/PRAK305-2410817210011-MUHAMMADDZULFATHIAHYAN.py
inputan = (int(input("")))
hari = (inputan / 86400)
jam = (inputan % 86400) / 3600
menit = (inputan % 3600) / 60
detik = (inputan % 60)
if hari >= 1:print("%02d hari %02d:%02d:%02d\n"%(hari,jam,menit,detik))
else : print("%02d:%02d:%02d\n"%(jam,menit,detik))